### PR TITLE
Add Takeshi Yamamuro to the list of committers

### PR DESCRIPTION
**Pull request by maropu**

### CODE REVIEW

**Comment by maropu** [comment]
cc: @gatorsmile

**Review by HyukjinKwon — APPROVED** [review]
Wow!

**Comment by maropu** [comment]
@HyukjinKwon ;))

**Comment by HyukjinKwon** [comment]
Hey @maropu, can you try to merge this one by yourself? or should I?

File: committers.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ navigation:
 |Patrick Wendell|Databricks|
 |Andrew Xia|Alibaba|
 |Reynold Xin|Databricks|
+|Takeshi Yamamuro|NTT|
```
Comment (HyukjinKwon) [inline committers.md:74]: Oh btw you should generate html too

**Comment by HyukjinKwon** [comment]
Also Shane missed the html generation. Let's add him in the html as well.

**Comment by maropu** [comment]
oh, I will.

**Comment by maropu** [comment]
Can you merge this? (I'm still not sure how to...)

**Comment by maropu** [comment]
fixed

**Comment by HyukjinKwon** [comment]
Here's my remote:

```
$ git remote -v
apache	https://git-wip-us.apache.org/repos/asf/spark-website.git (fetch)
apache	https://git-wip-us.apache.org/repos/asf/spark-website.git (push)
apache-github	https://github.com/apache/spark.git (fetch)
apache-github	https://github.com/apache/spark.git (push)
origin	git@github.com:HyukjinKwon/spark-website.git (fetch)
origin	git@github.com:HyukjinKwon/spark-website.git (push)
upstream	https://github.com/apache/spark-website.git (fetch)
upstream	https://github.com/apache/spark-website.git (push)
```

Set up the remote and then try to run the script:

```
$ ./merge_pr.py
```

Or, you can do it manually, for instance, 

```
$ git push apache <hash>:asf-site
```

When you do it manually, you can `git commit --amend` and add `Closes #146` at the end.

**Comment by maropu** [comment]
@HyukjinKwon thanks for your explanation! I'll check and try.

**Comment by maropu** [comment]
@HyukjinKwon ok, thanks! done.

**Comment by gatorsmile** [comment]
@maropu Congratulations!

**Comment by dongjoon-hyun** [comment]
Congratulation, @maropu ! :D

**Comment by maropu** [comment]
@gatorsmile @dongjoon-hyun Many thanks!

**Pull request closed without merging**